bsip-0048.md

```diff
@@ -10,45 +10,61 @@
 
 # Abstract
 
-The possibility of creating assets on the BitShares Blockchain is a core feature
-of our platform. In the case of an ico, the ico-holders can create an asset with
-a fixed maximum supply. However this maximum supply can be changed by the
-asset creator (issuer) anytime. This exposes participators ico to the risk,
-of loosing value of their bought asset.
+The possibility of creating assets on the BitShares Blockchain is a core feature of our platform. However some things are too maleable when they should not be. Other things are inflexible when they should be.
 
-With this BSIP a flag will be introduced which forbids the modification of the maximum
-supply of an issuer created asset.
+In the case of an ICO, the ICO holders can create an asset with a maximum supply. However this maximum supply can be changed by the asset creator (issuer) anytime. This exposes participants in the ICO to the risk of losing value of their purchased asset.
+
+With this BSIP a flag will be introduced which forbids the modification of the maximum supply of an issuer created asset.
+
+In addition, another new flag will be introduced that prevents any more tokens from being issued in the future.
+
+And finally, the ability to change the precision of an asset will now be available as long as current supply is zero.
 
 # Motivation
 
-The implementation of a flag, to disable the modification of the maximum supply for 
-issuer created asset, has effects for both sides. On the one side the security for the
-participators or non-issuer users increases, because the value of their assets (tokens)
-can't be manipulated by exposing the market to more created tokens. And on the other side
-it is good marketing for the ico-hoders, because they can advertise with their fixed
-maximum supply.
+A flag to disable the modification of the maximum supply for issuer created asset will be implemented. This has benefits for both the participants and the ICO holder. Participants in the ICO are assured the value of their assets (tokens) will not be diluted by issuers creating more tokens. ICO holders are able to market the fact that the maximum supply is fixed.
```

Comment:
Better "token holder" not "ICO holder". "ICO" is just one example where this might be important.

Reply:
Oh, you mean "ICO holder" as the entity who make the offer. Then perhaps "asset issuer". "ICO" is still just an example.

Reply:
I went through the document and attempted to standardize the language. Please let me know if it is now clearer.

Reply:
Good. Perhaps add a sentence about the ICO example use-case?

Reply:
@pmconrad  Please take a look at new line 27. Let me know if this is what you're looking for. Thanks for your input!